outputs.tf

```diff
@@ -31,12 +31,52 @@ output "security_group_id" {
   value       = "${aws_security_group.main.id}"
 }
 
+output "subnet_group_arn" {
+  description = "The ARN of the db subnet group."
+  value       = "${aws_db_subnet_group.main.arn}"
+}
+
 output "subnet_group_id" {
   description = "The db subnet group name."
   value       = "${aws_db_subnet_group.main.id}"
 }
 
-output "subnet_group_arn" {
-  description = "The ARN of the db subnet group."
-  value       = "${aws_db_subnet_group.main.arn}"
+output "address" {
+  description = "The address of the RDS instance"
+  value       = "${aws_db_instance.main.address}"
+}
+
+output "arn" {
+  description = "The ARN of the RDS instance"
+  value       = "${aws_db_instance.main.arn}"
+}
+
+output "availability_zone" {
+  description = "The availability zone of the RDS instance"
+  value       = "${aws_db_instance.main.availability_zone}"
+}
+
+output "zone_id" {
+  description = "The canonical hosted zone ID of the DB instance (to be used in a Route 53 Alias record)"
+  value       = "${aws_db_instance.main.hosted_zone_id}"
+}
+
+output "resource_id" {
+  description = "The RDS Resource ID of this instance"
+  value       = "${aws_db_instance.main.resource_id}"
+}
+
+output "status" {
+  description = "The RDS instance status"
+  value       = "${aws_db_instance.main.status}"
+}
+
+output "username" {
+  description = "The master username for the database"
+  value       = "${aws_db_instance.main.username}"
+}
+
+output "password" {
```

Comment:
I don't think we want to output this since the password could end up in CI/CD logs and we already know the password since its a required input variable.

Reply:
Well, it is referenced in old module in the outputs. To keep some compatibility, I have kept it in outputs of a new module for `neo-rds` (this PR https://github.com/TeliaSoneraNorge/devops/pull/493):
```
output "this_db_instance_password" {
  description = "The database password (this password may be old, because Terraform doesn't track it after initial creation)"
  value       = "${module.rds.password}"
}
```
From the description, I guess it is not needed. Password is generated anyway and saved as `aws_ssm_parameter`.